src/metabase/lib/remove_replace.cljc

```diff
@@ -134,7 +135,7 @@
     query))
 
 (defn- remove-replace* [query stage-number target-clause remove-or-replace replacement]
-  (binding [mu/*enforce* false]
+  (binding #?(:clj [mu.fn/*enforce* false] :cljs  [])
```

Comment:
can this be turned into a macro so callers don't need to remember this and the `require`?

Reply:
yeah, this is a pretty icky hack, I want to move `*enforce*` back into a Cljs namespace before this PR lands

Reply:
I added a `mu/disable-enforcement` macro for this